packages/bot-web-ui/src/components/dashboard/bot-builder/bot-builder.tsx

```diff
@@ -5,10 +5,10 @@ import { observer } from '@deriv/stores';
 import { useDBotStore } from '../../../stores/useDBotStore';
 import LoadModal from '../../load-modal';
 import SaveModal from '../dashboard-component/load-bot-preview/save-modal';
-import { BOT_BUILDER_TOUR } from '../joyride-config';
+import { BOT_BUILDER_TOUR } from '../dbot-tours/config';
+import MobileTours from '../dbot-tours/mobile-tours/mobile-tours';
```

Comment:
It should be .../mobile-tours instead of mobile-tours/mobile-tours @rupato-deriv